src/mocks.ts

```diff
@@ -2,10 +2,11 @@ import * as jsdom from 'jsdom';
 import { Cookie } from 'tough-cookie';
 import { UID2 } from './uid2Sdk';
 import { Uid2Identity } from './Uid2Identity';
-import { localStorageKeyName } from './uid2LocalStorageManager';
-import { base64ToBytes, bytesToBase64 } from './uid2Base64';
+import { base64ToBytes, bytesToBase64 } from './encoding/uid2Base64';
 import * as crypto from 'crypto';
 
+const uid2LocalStorageKeyName = 'UID2-sdk-identity';
```

Comment:
I didn't want to expose this value within the SDK itself as consumers shouldn't touch the local storage value - putting it directly in the test seemed reasonable. It also makes sure we're actually putting it in the right place - we can check EUID as well and make sure each one is going to the right place.